src/components/common/modal/ModalLayout.jsx

```diff
@@ -0,0 +1,32 @@
+import * as S from './ModalLayout.styled';
+import close from '../../../assets/icon/modal_close.svg'
+import useModal from "../../../hooks/useModal";
+import useModalScrollBlock from '../../../hooks/useMoadlScrollBlock'
+
+const ModalLayout = (props) => {
```

Comment:
아래 계속 props.으로 참조하고 있는데 여기서 구조분해 할당해서 들어가면 props. 안붙여도 됩니다!
```suggestion
const ModalLayout = ({title, subTitle, children}) => {
```